spec/services/container_deployment_service_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'recursive-open-struct'
+
+describe ContainerDeploymentService do
+  before do
+    %w(amazon openstack google azure redhat vmware).each do |p|
+      network = FactoryGirl.create(:network, :ipaddress => "127.0.0.1")
+      hardware = FactoryGirl.create(:hardware)
+      hardware.networks << network
+      vm = FactoryGirl.create("vm_#{p}".to_sym, :hardware => hardware)
+      if (p.include? "amazon") || (p.include? "redhat")
```

Comment:
you can replace the line 
`if (p.include? "amazon") || (p.include? "redhat")`
with the line
`if ['amazon', 'redhat'].include?(p) `
i think it looks better :cat2: and it does the same thing.